integration/spark2/src/main/scala/org/apache/spark/sql/execution/command/carbonTableSchema.scala

```diff
@@ -344,6 +345,13 @@ case class CreateTable(cm: TableModel, createDSTable: Boolean = true) extends Ru
 
     val tableInfo: TableInfo = TableNewProcessor(cm)
 
+    // Add validation for sort scope when create table
+    val sortScope = tableInfo.getFactTable.getTableProperties.get("sort_scope")
+    if (null != sortScope && !CarbonUtil.isValidSortOption(sortScope)) {
```

Comment:
null != sortScope can be removed, it is checked inside CarbonUtil.isValidSortOption

Reply:
ok, I will provide default value instead of null, then I will remove this